docs/managed-schema.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Managed schemas
+---
+
+Record-Store is great to handle customers defined schema, but sometimes you may need to store something simple.
+
+We are now providing dedicated `recordSpace` where the schema is directly handled by the Record-Store, lowering the difficulty to use the Record-Store. Under the hood, you will still open a `recordSpace`, but you will not be able to manipulate the schema.
```

Comment:
Remove temporality (now). Would be better adopting a ligher style (we are providing => A recordSpace is a keyspace directly managed by... )